api/lambdas/sqs-to-converter/index.js

```diff
@@ -116,7 +116,7 @@ function postProcessSidechainFHIRBundle(conversionResult, extension) {
       }
 
       if (idToUse) {
-        if (!uuid.validate(idToUse)) {
+        if (!uuid.validate(idToUse) && !(idToUse === patientId)) {
```

Comment:
Defs NABD now, but `!==` is easier to read, no need to negate the result of `===`

Reply:
agreed and fixed